src/ResourceManager/AzureBackup/Commands.AzureBackup/Cmdlets/Backup/BackupAzureRMBackupItem.cs

```diff
@@ -12,17 +12,19 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.AzureBackup.Models;
-using Microsoft.Azure.Commands.AzureBackup.Properties;
 using System;
 using System.Linq;
 using System.Management.Automation;
+using Microsoft.Azure.Commands.AzureBackup.Models;
+using Microsoft.Azure.Commands.AzureBackup.Properties;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.AzureBackup.Cmdlets
 {
     /// <summary>
     /// Backup Azue Backup Item
     /// </summary>
+    [CmdletDeprecation()]
```

Comment:
@dragonfly91 please add a message in these attributes to alert users that the module will be deprecated and they should use `AzureRM.RecoveryServices.Backup` (or `Az.RecoveryServices.Backup`) moving forward